prestoadmin/plugin.py

```diff
@@ -47,3 +46,15 @@ def add_jar(local_path, plugin_name, plugin_dir=REMOTE_PLUGIN_DIR):
     """
     _LOGGER.info('deploying jars on %s' % env.host)
     write(local_path, os.path.join(plugin_dir, plugin_name))
+
+def remove_jar(local_path, plugin_name):
+    """
+    Remove jar for the specified plugin to the plugin directory.
+
+    Parameters:
+        local_path - Local path to the jar to be removed
+        plugin_name - Name of the plugin subdirectory to deploy jars to
+
+    """
+    _LOGGER.info('removing jar on %s' % env.host)
+    os.remove(os.path.join(local_path, plugin_name))
```

Comment:
This will only remove the jar from the local system the presto-admin is running on.  To remove it from all of the presto nodes you should use a fabric command something like.

path = os.path.join(plugin_dir, plugin_name)
sudo('rm %s' % path)